Formula/c/csound.rb

```diff
@@ -21,6 +21,23 @@ class Csound < Formula
       url "https://github.com/csound/csound/commit/2a071ae8ca89bc21b5c80037f8c95a01bb670ac9.patch?full_index=1"
       sha256 "c7026330b5c89ab399e74aff17019067705011b7e35b9c75f9ed1a5878f53b4b"
     end
+
+    # Fix build failure due to incorrect member name on macOS 15+
```

Comment:
Where does this patch come from? Has it been upstreamed?

Reply:
I wrote this patch thinking it wasn’t available upstream, but I found it ([here](https://github.com/csound/csound/commit/bb9bafcfa17a87d3733eda1e25a812fd0be08ac6)) after some more digging (it wasn’t on the branch I was originally searching). I’ve updated this PR to use the upstream commit; apologies for the duplicative commits here.